From 3efd0909a5aa8b5bcd76defc8d1d3e2254be61dc Mon Sep 17 00:00:00 2001
From: Cristian Iorga <cristian.iorga@intel.com>
Date: Wed, 30 Mar 2016 20:14:36 +0300
Subject: [PATCH 3/3] Fix Linux libbt-vendor build

Fix enumeration values not explicitly handled in switch;
Fix the undefined reference to functions from libosi.

Change-Id: I64ab3d57d1c4879ad47ae1a03f8996287877535d
Signed-off-by: Cristian Iorga <cristian.iorga@intel.com>
---
 vendor_libs/linux/Android.mk        | 2 ++
 vendor_libs/linux/bt_vendor_linux.c | 6 ++++++
 2 files changed, 8 insertions(+)

diff --git a/vendor_libs/linux/Android.mk b/vendor_libs/linux/Android.mk
index c5b120d..760a698 100644
--- a/vendor_libs/linux/Android.mk
+++ b/vendor_libs/linux/Android.mk
@@ -31,6 +31,8 @@ LOCAL_C_INCLUDES += \
 LOCAL_SHARED_LIBRARIES := \
         libcutils

+LOCAL_STATIC_LIBRARIES := libosi
+
 LOCAL_MODULE := libbt-vendor
 LOCAL_MODULE_TAGS := optional

diff --git a/vendor_libs/linux/bt_vendor_linux.c b/vendor_libs/linux/bt_vendor_linux.c
index a07e313..16ecaec 100644
--- a/vendor_libs/linux/bt_vendor_linux.c
+++ b/vendor_libs/linux/bt_vendor_linux.c
@@ -402,6 +402,12 @@ static int bt_vendor_op(bt_vendor_opcode_t opcode, void *param)
   case BT_VND_OP_EPILOG:
     bt_vendor_callbacks->epilog_cb(BT_VND_OP_RESULT_SUCCESS);
     break;
+
+  case BT_VND_OP_A2DP_OFFLOAD_START:
+    break;
+
+  case BT_VND_OP_A2DP_OFFLOAD_STOP:
+    break;
   }

   LOG_INFO(LOG_TAG, "%s op %d retval %d", __func__, opcode, retval);
--
2.7.4
